cypress/e2e/signIn.cy.js

```diff
@@ -29,6 +31,12 @@ describe('Sign In page', () => {
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    signInPage.visit();
+
+    signInPage.typeEmail('wrongemail@example.com');
+    signInPage.typePassword('WrongPassword123!');
```

Comment:
Ensure that the `typePassword` method in `SignInPageObject` is correctly implemented to input the wrong password.